templates/careers/company-culture/identity.html

```diff
@@ -0,0 +1,82 @@
+<section class="p-strip is-shallow u-no-padding--bottom">
+    <div class="row--50-50 p-section">
+      <div class="col">
+        <h1 class="p-heading--2"><strong>Personal identity <br/> at Canonical</strong></h1>
+      </div>
+      <div class="col">
+        <p>At Canonical we believe your identity has no intrinsic bearing on your ability. As humans, we all have elements of our identity which we did not choose. Our name. Our skin colour. Our nationality of birth. Our sexuality. Our gender. We may be neurodiverse, or have a disability. Some of these attributes may not be obvious.</p>
+        <p> Whatever your identity, you are welcome as a colleague if you share our values, our mission, our work ethic and our skills.</p>
+        <p>We have many outstanding colleagues who represent a wide spectrum of humanity. We treasure their contributions, and we stand for their right to pursue their ambitions as equals in the company.</p>
+        <p>We explicitly choose to reject discriminatory views which link identity with ability. Those who espouse such views are unwelcome at Canonical.</p>
+        <p>We work as a global team to ensure this value is consistently true. We leave no grey areas for equivocation. We expect each and every member of the company to stand clearly behind this position, and to share the responsibility of removing those who undermine it.</p>
+        <p>The Ubuntu Code of Conduct is a founding document for the company. It says:</p>
+        <div class="p-strip">        
+          <div class="p-highlight--vertical">
+            <p class="p-heading--5">Ubuntu is about showing humanity to one another: the word itself captures the spirit of being human.</p>
+            <p>We want a productive, happy and agile community that can welcome new ideas in a complex field, improve every process every year, and foster collaboration between groups with very different needs, interests and skills.</p>
+            <p>We gain strength from diversity, and actively seek participation from those who enhance it. This code of conduct exists to ensure that diverse groups collaborate to mutual advantage and enjoyment. We will challenge prejudice that could jeopardise the participation of any person in the project.</p>
+          </div>
+        </div>
+        <p>This statement is as true of Canonical as it is of the Ubuntu project.</p>
+        <p>We did not set out to create the world’s most profitable software company at any cost. We set out to bring the benefits of open source to all humanity by making it easier and cheaper to consume and easier to improve, to level the playing field for innovators, entrepreneurs, students and digital consumers all across the globe.</p>
+        <p>We also set out to create a productive work environment for those who want to contribute to the open source revolution we are creating - living and working wherever they prefer, regardless of their identity. We continue to seek ways to improve on these fronts.</p>
```

Comment:
```suggestion
        <p>We also set out to create a productive work environment for those who want to contribute to the open source revolution we are creating &ndash; living and working wherever they prefer, regardless of their identity. We continue to seek ways to improve on these fronts.</p>
```
For the ones below as well